plugins/jobs/src/js/components/form/helpers/JobParsers.ts

```diff
@@ -44,16 +80,41 @@ export const jobSpecToFormOutputParser = (jobSpec: JobSpec): FormOutput => {
         jobSpec.job.run.ucr.image &&
         jobSpec.job.run.ucr.image.id
       : jobSpec.job.run.docker && jobSpec.job.run.docker.image;
+  const dockerParameters =
+    jobSpec.job.run.docker &&
+    Array.isArray(jobSpec.job.run.docker.parameters) &&
+    jobSpec.job.run.docker.parameters.length > 0
+      ? jobSpec.job.run.docker.parameters
+      : [];
+  const args =
+    Array.isArray(jobSpec.job.run.args) && jobSpec.job.run.args.length > 0
+      ? jobSpec.job.run.args
+      : [];
+  const imageForcePull =
+    container === Container.UCR
+      ? jobSpec.job.run.ucr &&
```

Comment:
We have an util for this - https://github.com/dcos/dcos-ui/blob/master/src/js/utils/Util.js#L54 .